backend/spindle/views.py

```diff
@@ -7,51 +8,81 @@
 import base64
 
 
+class SpindleError(Enum):
```

Comment:
I would suggest a different name for this class because `SpindleError` suggests it is an exception